quantmsrescore/ms2_model_manager.py

```
@@ -220,14 +325,14 @@ class MS2pDeepModel(pDeepModel):
 
     def __init__(
-        self,
-        charged_frag_types=get_charged_frag_types(frag_types, max_frag_charge),
-        dropout=0.1,
-        model_class: torch.nn.Module = ModelMS2Bert,
-        device: str = "gpu",
-        mask_modloss: Optional[bool] = None,
-        override_from_weights: bool = False,
-        **kwargs,  # model params
+            self,
+            charged_frag_types=get_charged_frag_types(frag_types, max_frag_charge),
```

Ruff (0.14.10)

Do not perform function call `get_charged_frag_types` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)